src/main/java/athleticli/ui/Parser.java

```diff
@@ -110,16 +222,19 @@ public static Command parseCommand(String rawUserInput) throws AthletiException
      * @throws AthletiException
      */
     public static Activity parseActivity(String arguments) throws AthletiException {
-        final int durationIndex = arguments.indexOf("duration/");
-        final int distanceIndex = arguments.indexOf("distance/");
-        final int datetimeIndex = arguments.indexOf("datetime/");
+        final int durationIndex = arguments.indexOf(Parameter.DURATION_SEPARATOR);
+        final int distanceIndex = arguments.indexOf(Parameter.DISTANCE_SEPARATOR);
+        final int datetimeIndex = arguments.indexOf(Parameter.DATETIME_SEPARATOR);
 
         checkMissingActivityArguments(durationIndex, distanceIndex, datetimeIndex);
 
         final String caption = arguments.substring(0, durationIndex).trim();
-        final String duration = arguments.substring(durationIndex + 9, distanceIndex).trim();
-        final String distance = arguments.substring(distanceIndex + 9, datetimeIndex).trim();
-        final String datetime = arguments.substring(datetimeIndex + 9).trim();
+        final String duration =
+                arguments.substring(durationIndex + Parameter.DURATION_SEPARATOR.length(), distanceIndex).trim();
+        final String distance =
+                arguments.substring(distanceIndex + Parameter.DISTANCE_SEPARATOR.length(), datetimeIndex).trim();
+        final String datetime =
+                arguments.substring(datetimeIndex + Parameter.DATETIME_SEPARATOR.length()).trim();
```

Comment:
Great use of named constants for better maintainability and readability, also prevents bugs in the future when parameters are changed